### PR TITLE
BZ-1278722 - Marshalling Error when executing command through SOAP interface

### PR DESCRIPTION
**Pull request by mrietveld**

### CODE REVIEW

File: kie-remote/kie-remote-services/src/test/java/org/kie/remote/services/rest/query/QueryResourceQueryHelperTest.java [hunk 1]
```diff
@@ -41,6 +41,7 @@
 import org.kie.api.task.model.TaskSummary;
 import org.kie.internal.identity.IdentityProvider;
 import org.kie.internal.task.api.InternalTaskService;
+import org.kie.remote.services.RandomUtil;
```
Comment (mswiderski) [inline kie-remote/kie-remote-services/src/test/java/org/kie/remote/services/rest/query/QueryResourceQueryHelperTest.java]: looks like dead import

File: kie-remote/kie-remote-services/src/test/java/org/kie/remote/services/rest/query/QueryResourceTest.java [hunk 2]
```diff
@@ -431,6 +432,8 @@ public void testTaskSummary() {
         List<TaskSummary> taskSumList = resp.getResult();
         assertNotNull( "Null task summary list", taskSumList );
         assertFalse( "Empty task summary list", taskSumList.isEmpty() );
+
+        Set<Long> uniqueTaskIds = new HashSet<Long>();
```
Comment (mswiderski) [inline kie-remote/kie-remote-services/src/test/java/org/kie/remote/services/rest/query/QueryResourceTest.java]: doesn't seem to be used anywhere...

**Comment by mswiderski** [comment]
+1 to merge - just tiny two cleanup comments

**Comment by mrietveld** [comment]
Code cleaned up, thanks!

**Comment by mrietveld** [comment]
Jenkins, please retest.

**Comment by mrietveld** [comment]
@mswiderski If you get a chance, could you merge this?

**Comment by mswiderski** [comment]
merged into 6.4.x

**Pull request closed without merging**